### PR TITLE
feat: invite emails from assignment

### PR DESCRIPTION
**Pull request by ibex088**
## What does this PR do?

Fixes #13532
/claim #13532

[loom](https://www.loom.com/share/e22d9acd73544a279ba59d1bc4d832b4?sid=2b64d6f3-fb06-485b-a5c7-460bfa3b8037)

ps: have disabled the pending users in dropdown, because pending users dont have their Availability set-up.

### CODE REVIEW

**Comment by supalarry** [comment]
1. Is it hard to allow adding existing team members by e-mail? I think that if we can add new members by e-mail, we should allow searching existing ones by e-mail too.
2. I think under "Team" heading above dropdown to "Add attendees" should be an explanation - "Add existing team members by name or e-mail or invite new team members using e-mail" because otherwise this feature is hidden from users.

@PeerRich do you agree?

**Comment by joeauyeung** [comment]
> 1. Is it hard to allow adding existing team members not only by name but also by e-mail? I think that if we can add new members by e-mail, we should allow searching existing ones by e-mail too.
> 2. I think under "Team" heading above dropdown to "Add attendees" should be an explanation - "Add existing team members by name or e-mail or invite new team members using e-mail" because otherwise this feature is hidden from users.
> 
> @PeerRich do you agree?

What if under the user dropdown link we add an option "Invite new team members" or something similar that opens the member invite dialog. That way we can reuse the logic or email or username look up and role assignment. I think the only thing that needs to be added would be to auto assign them to the team but not included in the availability query (might already exist?)

**Review by CarinaWolli — CHANGES_REQUESTED** [review]
Can you please address my comments below, let me know if you have any questions 
🙏

Also, there are merge conflicts. Most of them are probably coming from this PR: https://github.com/calcom/cal.com/pull/13566

**Comment by SomayChauhan** [comment]
@CarinaWolli 
are we sure we want to go forward with this approach
i mean @joeauyeung suggestion was pretty good too

**Comment by CarinaWolli** [comment]
> What if under the user dropdown link we add an option "Invite new team members" or something similar that opens the member invite dialog. That way we can reuse the logic or email or username look up and role assignment. I think the only thing that needs to be added would be to auto assign them to the team but not included in the availability query (might already exist?)

Sorry, I totally missed that comment. I do think we should make it more clear, like @supalarry said, right now this feature would be hidden for users. @joeauyeung where would you put the option to 'Invite new team members'? I think we want to avoid to add anything below the list of hosts, as this list can get long

**Comment by CarinaWolli** [comment]
Maybe @ciaranha can give some input here how to best do that ui/ux-wise

**Comment by CarinaWolli** [comment]
@SomayChauhan Is this the same as https://github.com/calcom/cal.com/pull/14185?

**Comment by SomayChauhan** [comment]
yes!!, closing this in favour of https://github.com/calcom/cal.com/pull/14185

File: packages/features/eventtypes/components/CheckedTeamSelect.tsx [hunk 1]
```diff
@@ -33,6 +104,34 @@ export const CheckedTeamSelect = ({
...
         value={value}
+        onBlur={() => {
+          setError(null);
+        }}
+        onInputChange={(e) => {
+          const targetValues = e.split(/[\n,]/);
+          const emails =
+            targetValues.length === 1
+              ? targetValues[0].trim().toLocaleLowerCase()
+              : targetValues.map((email) => email.trim().toLocaleLowerCase());
```
Comment (hamzakhalidchouhdary) [inline packages/features/eventtypes/components/CheckedTeamSelect.tsx:115]: `const emails = targetValues.map((email) => email.trim().toLocaleLowerCase());`

**Pull request closed without merging**